### ISSUE TITLE
BO (Bolivia) flag error

### ISSUE THREAD

**Issue by maLikiz** [catamphetamine/react-phone-number-input#258]
I'm trying to use a list of flags, and I get an error using the BO flag and not only

```javascript
import React, { Component } from 'react';
import flags from 'react-phone-number-input/flags';

export default SimpleClass extends Component {
  render() {
    return flags.BO()
  }
}
```

Errors:
Uncaught Invariant Violation: The `style` prop expects a mapping from style properties to values, not a string. For example, style={{marginRight: spacing + 'em'}} when using JSX.

Warning: Invalid DOM property `class`. Did you mean `className`?

The above error occurred in the <path> component:

**Comment by maLikiz** [comment]
But not reproduced if I use `countrySelectComponent`:
```javascript
<PhoneInputLibrary
    countrySelectComponent={props =>
      props.options.map((item, index) => {
        // I don't understand why ref argument is needed:
        item.icon(item);
      })
    }
  />
```

**Comment by catamphetamine** [comment]
Hmm, indeed, `flags.js` contains many `style: string` properties.

```xml
<path style="marker:none"/>
<path fontSize="100" style="line-height:125%;-inkscape-font-specification:'Linux Biolinum Bold';text-align:center" fill="#e8a30e" stroke="#000" />
<path style="line-height:normal;text-indent:0;text-align:start;text-decoration-line:none;text-decoration-style:solid;text-decoration-color:#000;text-transform:none;block-progression:tb;isolation:auto;mix-blend-mode:normal" />
```

This should be somehow transformed into objects I guess.

**Comment by catamphetamine** [comment]
Fixed in `react-phone-number-input@2.3.18`: it uses `svgr` for transforming SVG to React.
